website/_sass/_base.scss

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
scss is "sassy css" a sass/css dialect. This is all Jekyll default stuff and fine as-is